packages/web-components/src/presence-badge/fixtures/presence-badge.html

```diff
@@ -0,0 +1,562 @@
+<style>
+  fluent-design-system-provider {
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans',
+      'Helvetica Neue', sans-serif;
+    padding: 20px;
+  }
+
+  div > fluent-presence-badge {
+    margin-right: 20px;
+  }
+
+  div > fluent-presence-badge:last-child {
+    margin-right: 0px;
+  }
+
+  dl {
+    display: grid;
+    grid-template-columns: max-content auto;
+  }
+
+  dt {
+    grid-column-start: 1;
+  }
+
+  dd {
+    grid-column-start: 2;
+    margin-inline-start: 0.75rem;
+    margin-bottom: 1rem;
+  }
+
+  div.presence-holder {
+    background-image: linear-gradient(to top, #ddd 0px, #ddd 16px, rgba(0, 0, 0, 0) 16px, rgba(0, 0, 0, 0) 100%);
+  }
+
+  table {
+    margin: 5vw;
+  }
+
+  caption {
+    text-align: left;
+    margin-bottom: 8px;
+  }
+
+  tr td {
+    border-bottom: 1px solid #eee;
+  }
+
+  tr:last-of-type td {
+    border-bottom: none;
+  }
+
+  th {
+    font-weight: 300;
+    font-size: 0.8rem;
+    vertical-align: bottom;
+    padding-bottom: 12px;
+  }
+
+  th[scope='row'] {
+    text-align: right;
+    vertical-align: middle;
+    padding-right: 8px;
+    font-size: 0.6rem;
+    margin-bottom: 0;
+  }
+
+  td {
+    text-align: center;
+    padding: 11px 1vw;
+    min-width: 0.6rem;
+    width: 11%;
+  }
+
+  .sr-only {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
+    border: 0;
+  }
+
+  fluent-design-system-provider > div {
+    display: block;
+    clear: both;
+    float: left;
+    border-radius: 8px;
+    background: #fafafa;
+    margin: 0 0 26px;
+  }
+
+  fluent-design-system-provider > div.padded {
+    padding: 10px 32px;
+  }
+
+  @media (max-width: 564px) and (orientation: portrait) {
+    fluent-design-system-provider {
+      padding: 4px;
+    }
+
+    th {
+      font-size: 0.45rem;
+    }
+  }
+</style>
+<fluent-design-system-provider use-defaults style="background: #fefefe;">
```

Comment:
We should probably remove this from the sample since we don't require its usage anymore.